test-zuul/src/main/resources/application.yml

```diff
@@ -2,12 +2,19 @@ spring:
   application:
     name: ZUUL
 server:
+  address: 127.0.0.1
   port: 8765
 eureka:
   instance:
     metadataMap:
       instanceId: ${spring.application.name}-${random.number}
+    ip-address: 127.0.0.1
+    hostname: localhost
+    prefer-ip-address: false
   client:
     enabled: true
     serviceUrl:
-      defaultZone: http://${eureka.host:localhost}:${eureka.port:1111}/eureka/
\ No newline at end of file
+      defaultZone: http://${eureka.host:localhost}:${eureka.port:1111}/eureka/
+    use-dns-for-fetching-service-urls: false
+zuul:
+  sensitive-headers: Authorization
```

Comment:
Scott, thank you very much indeed. You've explained it properly and I could get it working, however, it still makes a redirection to the port in firefox, when user isn't already logged in. I've updated the project in order to enable the self preservation mode by default in Eureka and have the instance removed faster. Now I'm going to send an e-mail to the group in order to discuss the design flaw you talk about in your previous comments.